samples/client/petstore/typescript-node/default/api/petApi.ts

```diff
@@ -83,17 +83,17 @@ export class PetApi {
     /**
      * 
      * @summary Add a new pet to the store
-     * @param body Pet object that needs to be added to the store
+     * @param pet Pet object that needs to be added to the store
      */
-    public async addPet (body: Pet, options: {headers: {[name: string]: string}} = {headers: {}}) : Promise<{ response: http.ClientResponse; body?: any;  }> {
```

Comment:
this is unrelated

Reply:
Similar to above. Apparently something was landed earlier but the samples weren't updated (generated by the build script). I'm happy to revert this file back to master but shouldn't samples always be reflective of the latest version of the tool?

Reply:
I will update your branch with the correct samples